db/schema.rb

```diff
@@ -73,6 +72,7 @@
     t.string   "name",       null: false
     t.boolean  "billable",   null: false
     t.integer  "tock_id",    null: false
+    t.boolean  "live"
```

Comment:
I guess we could make it false? But we're getting this value from Tock, so I am unsure.